.changeset-staged/six-falcons-sin.md

```diff
@@ -0,0 +1,7 @@
+---
+"@logto/cli": minor
+"@logto/core": minor
+---
+
+- cli: use `ec` with `secp384r1` as the default key generation type
+- core: use `ES384` as the signing algorithm for EC keys
```

Comment:
I remember you said we need to commit the changeset in a separate PR, is that still true now?

Reply:
no, we should commit WITH the PR since we have this staged folder now